pkg/operator/ceph/file/filesystem_test.go

```diff
@@ -497,24 +485,14 @@ func TestUpgradeFilesystem(t *testing.T) {
 			versionStr, _ := json.Marshal(
 				map[string]map[string]int{
 					"mds": {
-						"ceph version 16.0.0-4-g2f728b9 (2f728b952cf293dd7f809ad8a0f5b5d040c43010) pacific (stable)": 2,
+						"ceph version 17.2.0-0-g2f728b9 (2f728b952cf293dd7f809ad8a0f5b5d040c43010) quincy (stable)": 2,
 					},
 				})
 			return string(versionStr), nil
 		}
 		assert.Fail(t, fmt.Sprintf("Unexpected command %q %q", command, args))
 		return "", nil
 	}
-	// do upgrade
-	clusterInfo.CephVersion = version.Quincy
-	context = &clusterd.Context{
-		Executor:  executor,
-		ConfigDir: configDir,
-		Clientset: clientset,
-	}
-	err = createFilesystem(context, clusterInfo, fs, &cephv1.ClusterSpec{}, ownerInfo, "/var/lib/rook/")
-	assert.Error(t, err)
-	assert.Contains(t, err.Error(), "fail mds failed")
```

Comment:
Don't we need to keep this?

Reply:
I thought an earlier step of this test already upgraded to quincy, but i'll take another look to confirm...

Reply:
Ok added it back, I see the intended error condition now.